api_docs/include/rest-endpoints.md

```diff
@@ -72,6 +72,7 @@
 * [Reactivate a user](/api/reactivate-user)
 * [Get a user's status](/api/get-user-status)
 * [Update your status](/api/update-status)
+* [Update user status if you are an admin](/api/update-status-from-admin)
```

Comment:
Let's do `[Update a user's status](/api/update-status-for-user)` for the URL here.